uploadNeuroDB/minc_insertion.pl

```diff
@@ -538,11 +538,26 @@ =head2 Methods
         $subjectIDsref, 
         $studyInfo{'DateAcquired'},
         \$dbh,
-        $subjectIDsref->{'subprojectID'},
         $db
 
-   );
-
+);
+ 
+$subjectIDsref->{'SubprojectID'} = NeuroDB::MRI::getObjective($subjectIDsref, $dbh);
```

Comment:
shouldn't there be a check for ProjectID here too?

Reply:
There is a call the `determineSubjectID` a couple of lines further up and this method will validate that a ProjectID was supplied, so no need to do this twice.

Reply:
I don't see where in `determineSubjectID` the `ProjectID` is validated. Could you point it to me? Thanks!